### DISCUSSION TITLE
Email Reply-To Option

### DISCUSSION THREAD

**Discussion by christoferd** [InvoicePlane/InvoicePlane#1088] [Ideas]
Email Reply-To Option would be nice to have.
I personally send my invoice emails from my shared hosting, from a different subdomain than that of my own email address. Would like them to be able to reply to an invoice email without it having to be redirected from the shared hosting server.

Nowadays Google and Microsoft servers not liking email forwarding so much.

**Comment by clockwiseq** [comment]
Agreed.  I'll add this as a feature request in v2

**Comment by nielsdrost7** [comment]
@christoferd can you explain what it would look like?
My invoiceplane is at invoicing.mycorporation.com
I'm sending you an email and my email address is receivables@corpinc.com ?
And I need to put a reply-to address in the email that I send?

I'm trying to paint a picture of what to build in V2
